src/main/java/io/pakland/mdas/githubstats/domain/service/GetOrganizationFromTeamName.java

```diff
@@ -0,0 +1,25 @@
+package io.pakland.mdas.githubstats.domain.service;
+
+import io.pakland.mdas.githubstats.domain.model.Organization;
+import io.pakland.mdas.githubstats.domain.model.Team;
+import io.pakland.mdas.githubstats.domain.ports.TeamRepository;
+
+import java.util.Optional;
+
+public class GetOrganizationFromTeamName {
```

Comment:
Shouldn't this be marked as a `@Service`? @mikededo

Reply:
Yes, thanks.